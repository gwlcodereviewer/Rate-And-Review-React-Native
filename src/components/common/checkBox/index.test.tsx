import React from 'react';
import renderer from 'react-test-renderer';
import CheckBoxCircle from '.';

let wrapper: renderer.ReactTestRenderer;

describe('render correctly', () => {
  wrapper = renderer.create(<CheckBoxCircle />);

  it('render correctly checkBoxCircle view', () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
