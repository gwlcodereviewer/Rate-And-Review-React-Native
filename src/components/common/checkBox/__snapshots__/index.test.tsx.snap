// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly render correctly checkBoxCircle view 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "height": 28.5,
      "justifyContent": "center",
      "opacity": 1,
      "width": 28.5,
    }
  }
>
  <View
    border="#D9DCE6"
    style={
      Array [
        Object {
          "borderColor": "#D9DCE6",
          "borderRadius": 12.5,
          "borderStyle": "solid",
          "borderWidth": 1,
        },
      ]
    }
  >
    <svg
      color="#fff"
      data-file-name="SvgCheckedIcon"
      fill="#fff"
      height={22.5}
      style={
        Array [
          Object {},
        ]
      }
      width={22.5}
    />
  </View>
</View>
`;
